torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -60,7 +60,9 @@ static IValue Table(
 std::pair<IValue, IValue> getFunctionTuple(
     const Module& module,
     const Function& func,
-    BackendDebugInfoRecorder& debug_info_recorder) {
+    BackendDebugInfoRecorder& debug_info_recorder,
+    const std::basic_string<char>& qn,
+    TypeNameUniquer& type_name_uniquer_) {
```

Comment:
BTW trailing _ is for private class member names.

Reply:
We are using a const reference of the `type_name_uniquer_` from `ScriptModuleSerializer` in these functions. I thought will have the same name making it easier to search and understand. Anyone who searches for `type_name_uniquer_` in this file can discover the usages. I understand this violates the naming convention of the variable. What do you recommend in this case?

Reply:
I think that’s a good reason but you can leave name with _ in a comment so if someone greps they can find it which is what I presume you want.